<launch>
<node pkg="optimal_control_interface" name="optimal_control_node" type="optimal_control_interface_node" output="screen" ns="drone_5">
        <param name="drone_id" value="5"/>
        <param name="solver_rate" value="1"/>
        <rosparam param="desired_wp">[6.75, -64.36, 11]</rosparam>
        <rosparam param="drones">[4, 5, 6]</rosparam>
        <rosparam param="desired_vel">[0, 0, 0]</rosparam> 
        <rosparam param="no_fly_zone">[-2.4, -36.5]</rosparam> 
        <rosparam param="target_final_pose">[-7,-46]</rosparam>
        <param name="priority" type="int" value="2"/>
    </node> 
    
</launch>
